### PR TITLE
removed create_at and updated_at colums as per Doc issue 333

### PR DESCRIPTION
**Pull request by csobier**
@iriberri , part one Docs issue#333- https://github.com/CartoDB/docs/issues/333.  Please sign-off if correct.

### CODE REVIEW

File: doc/API.md [hunk 1]
```diff
@@ -146,8 +144,6 @@ https://{account}.cartodb.com/api/v2/sql?format=GeoJSON&q=SELECT * FROM {table_n
         month: 10,
         day: "11",
         cartodb_id: 1,
```
Comment (iriberri) [inline doc/API.md]: This last comma should be out :-)

**Comment by csobier** [comment]
@iriberri , removed comma as instructed.  Good?

**Comment by iriberri** [comment]
Awesome! Thanks :)

**Comment by csobier** [comment]
@rochoa , Carla and I made a small doc update (related to [Docs issue#333](https://github.com/CartoDB/docs/issues/333)).  Since you are the owner of this repo, please merge this branch.

**Comment by rochoa** [comment]
Thanks for keeping docs clean & tidy! :smile:

**Pull request merged**